packages/help-center/src/styles.scss

```
@@ -0,0 +1,116 @@
+.help-center__container {
+	background-color: #fff;
+	color: #000;
+
+	&.is-desktop {
+		background: white;
+		border-radius: 2px;
+		box-shadow: 0 0 3px 0 rgba( 0, 0, 0, 0.25 );
+		cursor: default;
+		display: inline;
+		min-width: 300px;
+		position: fixed;
+		right: 16px;
+		top: 74px;
+		width: 20vw;
+		z-index: 9999;
+		transition: 0.5s ease;
+
+		.help-center__container-header {
+			cursor: pointer;
+			height: 50px;
```

The header needs less horizontal padding, I suggest the following.
```suggestion
			height: 50px;
			padding-left: 10px;
			// the x button has some white space on the right side, this compensates for that
			padding-right: 5px;
```